docs/workers.md

```diff
@@ -0,0 +1,78 @@
+[Table of contents](README.md#table-of-contents)
+
+# Workers
+
+This page list all the currently available workers on the cozy-stack. It
+describes their input arguments object. See the [jobs document](./jobs.md) to
+know more about the API context in which you can see how to use these
+arguments.
+
+## `sendmail` worker
+
+The `sendmail` worker can be used to send mail from the stack. It implies that
+the stack has properly configured an access to an SMTP server. You can see an
+example of configuration in the [cozy.dist.yaml](../cozy.dist.yaml) file at
+the root of this repository.
+
+`sendmail` options fields are the following:
+
+- `mode`: string specifying the mode of the send:
+    - `noreply` to send a notification mail to the user
+    - `from` to send a mail from the user
+- `to`: list of object `{name, email}` representing the addresses of the
+  recipients. (should not be used in `noreply` mode)
+- `subject`: string specifying the subject of the mail
+- `parts`: list of part objects listing representing the content parts of the
+  mail's body
+
+A part can be represented by two different object types:
+
+  - `type` string of the content type: either `text/html` or `text/plain`
+  - `body` string of the actual body content of the part
+
+or
+
+  - `template` string of the [html template](https://golang.org/pkg/html/template/)
+  - `values` object containing the key/values to fill the template with
```

Comment:
The template can only be used with html? I'd have put the values outside of the parts and use them in both the html and text parts. Maybe something like this:

```js
{
    "mode": "from",
    "to": [
        {"name": "John Doe 1", "email":"john1@doe"},
        {"name": "John Doe 2", "email":"john2@doe"}
    ],
    "subject": "Hey !",
    "parts": [
        {"type": "text/html", "template":"<h1>{{.Title}}</h1>"},
        {"type": "text/plain", "body": "{{.Title}} !"}
    ],
    "values": {"Title": "Hello!"}
}
```